ml-deep-dive/svm.md

```diff
@@ -0,0 +1,96 @@
+# Support Vector Machines
+Support vector machines (SVMs) are another subset of supervised learning methods that can be used for either classification or regression tasks. To find out what makes Support Vector Machines unique, and get a more expansive overview of SVMs, we would encourage students to follow the tutorial linked below:  
```

Comment:
I am not sure, is the link optional now or not? I would say yes, but if not I would put it into Optional in the bottom.

Reply:
Yes, still recommended that students go through this tutorial, it is *not optional*. All optional reading is in the end.

Reply:
Would not make it optional either, SVMs are too important imo